src/components/Section/Section.stories.tsx

```diff
@@ -17,6 +16,10 @@ export default {
 
 type Args = React.ComponentProps<typeof Section>;
 
+const HelpIcon = (
+  <Icon name="help" purpose="informative" size="1.375rem" title="help" />
+);
+
```

Comment:
I changed both the `titleBefore` and `titleAfter` stories to use an icon without a button so it's easier to see how much spacing is added by the component between the text and whatever's being inserted before/after it.

### Before
<img width="1444" alt="section component in storybook before this change" src="https://user-images.githubusercontent.com/7761701/199843737-abd0e9ce-489b-4624-b276-3cb46d9bd1fb.png">

### After
<img width="1444" alt="section component in storybook after this change" src="https://user-images.githubusercontent.com/7761701/199843753-0733b1dc-85b1-45b2-9896-fca807b15818.png">